static/docs/commands-reference/get.md

```diff
@@ -18,19 +18,23 @@ positional arguments:
 
 ## Description
 
-In some cases it's convenient to get a <abbr>data artifact</abbr> from another
-DVC repository. The `dvc get` command helps the user do so. The `url` argument
-should provide the external DVC project's Git repository URL (both HTTP and SSH
-protocols supported, e.g. `[user@]server:project.git`), while `path` is used to
-specify the path to the data to be downloaded within the repo.
+DVC provides an easy way to reuse datasets, intermediate results, ML models, or
+other files and directories tracked in another DVC repository into the present
+<abbr>workspace</abbr>. The `dvc get` command downloads such a <abbr>data
```

Comment:
it's not even about present workspace only, it's about getting a file to deploy it for example. Or slice and dice with other tools, in an ad-hoc way. It not like you need a project/workspace for that.

Reply:
You're right. Updated in d53434d, check it out )